youtube_dl/extractor/webofstories.py

```diff
@@ -133,8 +133,8 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, playlist_id)
 
         entries = [
-            self.url_result('http://www.webofstories.com/play/%s' % video_number, 'WebOfStories')
-            for video_number in set(re.findall(r'href="/playAll/%s\?sId=(\d+)"' % playlist_id, webpage))
+            self.url_result('http://www.webofstories.com/play/%s/%d' % (playlist_id, video_number), 'WebOfStories')
+            for video_number in range(1, len(re.findall(r'class="playListItem\s+"', webpage)) + 2)
```

Comment:
Links must stay id based.

Reply:
Done. Currently the playlist items are of the following format:
`<a id="%d" class="title" title="%s">%s</a>`
The navigation is JS-event based, with no href defined on the element. I've modified the code to find based on existence of id attribute that contains a number, which seems to be how the page is laid out.

Reply:
Have you read my comment at all? Links **must stay id based**: https://www.webofstories.com/play/4538?o=MS, not https://www.webofstories.com/play/hans.bethe/73.